packages/components/src/tools-panel/types.ts

```diff
@@ -47,6 +52,10 @@ export type ToolsPanelProps = {
 	 * last visible `ToolsPanelItem` within the `ToolsPanel`.
 	 */
 	__experimentalLastVisibleItemClass?: string;
+	/**
+	 * The heading level of the panel's header.
+	 */
+	headingLevel: HeadingSize;
```

Comment:
I believe this prop should be marked as optional as assigned a default value, both for `ToolsPanel` and `ToolsPanelHeader`. Any reason why it was marked as required instead?